internal/run/run.go

```diff
@@ -853,3 +880,17 @@ var randStringBytes = func(n int) string {
 func getTempApplyPath(overlayFile string) string {
 	return filepath.Join(workflow.GetTempDir(), fmt.Sprintf("applied_%s%s", randStringBytes(10), filepath.Ext(overlayFile)))
 }
+
+func parseRegistryLocation(location workflow.SourcePublishLocation) (string, string, string, error) {
+	subParts := strings.Split(string(location), "registry.speakeasyapi.dev/")
```

Comment:
I feel like this belongs on `workflow.SourcePublishLocation` ?

Reply:
ya it probably should. I can add it